### DISCUSSION TITLE
请问显示时间格式错误怎么解决

### DISCUSSION THREAD

**Discussion by enohe** [mikumifa/biliTickerBuy#145] [提问专区]
![ORB3C@53MU 4IV7@GST~%23](https://github.com/mikumifa/biliTickerBuy/assets/173433814/99300a6d-f28b-4e81-b873-216266ff1950)
![_2Y~25 `O`@ZFK$%JS~JFCL](https://github.com/mikumifa/biliTickerBuy/assets/173433814/ec3a0788-96df-4972-ae54-c692e72f2a41)
![AP(QAAH%) ~Z1BP9XA_ { 0](https://github.com/mikumifa/biliTickerBuy/assets/173433814/1738f1b1-8130-4b11-b5ea-e40519fb945e)

**Comment by mikumifa** [comment]
选一下秒？

**Reply by enohe** [reply]
<img width="341" alt="屏幕截图 2024-06-21 130658" src="https://github.com/mikumifa/biliTickerBuy/assets/173433814/e14fcb3a-a326-40bd-9851-f9aba0a37688">
是有选的

**Reply by imRoxyy** [reply]
> 选一下秒？
现在可以选择00秒了吗

**Comment by mikumifa** [comment]
现在先选非00秒的时间，下一个版本会解决这个问题

**Reply by enohe** [reply]
![LWYK85 016%3@Y1 G@6IA%B](https://github.com/mikumifa/biliTickerBuy/assets/173433814/e71d4f2a-18d2-4ab7-a702-ac69f0dbab3a)
![NLJT}EWE4NMBF$}%H(ARZY](https://github.com/mikumifa/biliTickerBuy/assets/173433814/0d291ccf-24db-408a-bbbe-5c6e94b11809)
还有一个问题是按步骤做了，文件放到文件夹里但再打开还是这样显示，有什么影响吗?

**Reply by mikumifa** [reply]
这个文件有没有对抢票没有任何影响
